R/get.installed_path.R

```diff
@@ -20,10 +20,21 @@ get.installed_path <- function(pkg, vrs) {
   for (pathk in pkg_search_paths) {
     # If directory and file exists, test if package is installed, if file does not exist; it is not installed anyway
     if (file.exists(pathk)) {
+      # See if there isa  package there
       df.pkg <- data.frame(installed.packages(lib.loc = pathk)) # Note: in {groundhogR} each package version gets a 'library' within the R version
       if (nrow(df.pkg) > 0) {
-        installed_path <- pathk
-        break
+        # If there is, check its version (is unlikely but possible for wrong version to get installed (e.g., wrong Published.date in DESCRIPTION
+        # Get installed version
+        installed_version <- subset(df.pkg, Package == pkg)$Version
+        # If different from desired, delete folder
+        if (installed_version != vrs) {
+          unlink(pathk, recursive = TRUE)
```

Comment:
Is it actually needed at all? I'll need to think about it but if we run `install.snowball()` with `force.install = TRUE`, it should be enough.

Context: it's always dangerous to remove files recursively. If for some reason, the path for `pathk` was wrong, we might end up important user files (worst case scenario, the whole content of their home).

Reply:
@urisohn, thoughts on this?

Otherwise, this is ready to merge. I'll add the last changes and do a quick last round of review and we should be ready to go

Reply:
I understand the value of avoiing the unlink(, recursive=TRUE)

Force-install is not enough on its own, users will not realize they got the wrong version installed.

Option 1: if there is a mistmatch it could trigger an error.
Something like 
if (installed_version != vrs) {
stop("We tried installed x, but we installed y, maybe try this or that...")

Option 2: we do a force-install from source
if (installed_version != vrs) {
#run install.packages() using the source file
}

I prefer option 2. It relies on R's trigger happy folder concnet replacement without adding the risk you identify.

What do you think?